usdigitalresponse/usdr-gost
# Hide dashboard stats, skip separate query for total count

### Ticket #1829 
## Description
1. Dashboard stats are hidden with the same feature flag that enables new table, let me know if it should be removed altogether.
2. I went to remove the total count and page totals, but I couldn't not set `total-rows` and still have the next & previous buttons work. Instead, I took a stab at adding total to the main query so the second one doesn't need to run. It works in my testing and `EXPLAIN ANALYZE` on the query shows the same cost as before. But let me know if I'm missing something fundamental here & I'll look for a different approach. 

## Screenshots / Demo Video

### Before - Paginator - shows 5224 results
<img width="543" alt="image" src="https://github.com/usdigitalresponse/usdr-gost/assets/6497366/512b03c8-6963-4761-8bbb-874c9b2abd6b">

### After - Paginator - shows 5224 results (expected)
<img width="555" alt="image" src="https://github.com/usdigitalresponse/usdr-gost/assets/6497366/b41dffe7-105f-4eda-ad57-da0d1e01d709">


### Before - Dashboard - Notice the `/dashboard` request
<img width="1920" alt="image" src="https://github.com/usdigitalresponse/usdr-gost/assets/6497366/d5f5ba54-5a72-4ccd-aa14-ccbd5bfc389b">

### After - Dashboard - Notice no `/dashboard` request
<img width="1916" alt="image" src="https://github.com/usdigitalresponse/usdr-gost/assets/6497366/55992996-5a26-4955-87b1-03b6bcf0f599">


## Testing
1. Confirm dashboard stats don't display when new Grants table is on
2. Try different filters on grants search, total count should be the same as before. 

### Automated and Unit Tests

### Manual tests for Reviewer

## Checklist
